packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -242,17 +243,28 @@ const generatePermutation = (count: number, user: UsersCurrent|null): Array<numb
 
 
 const ReviewVotingPage = ({classes}: {
-  classes: ClassesType,
+  classes: ClassesType
 }) => {
+  const { LWTooltip, Loading, ReviewVotingExpandedPost, ReviewVoteTableRow, SectionTitle, RecentComments, FrontpageReviewWidget, ContentStyles, Error404 } = Components
+
   const currentUser = useCurrentUser()
   const { captureEvent } = useTracking({eventType: "reviewVotingEvent"})
-  
+  const { params, query } = useLocation()
+  const reviewYear = getReviewYearFromString(params.year)
+  if (!reviewYear) return <Error404 />
```

Comment:
Need to either move this after all the hooks (since hooks can't be called conditionally).  But if you want to avoid making extra network calls, you can still check the validity of the query param here and if it's not valid, set the `skip` value in `useMulti` based on that.